events.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+    "name": "Paris December 7",
+    "slug": "graphql-paris-pre-apidays",
+    "location": "Paris",
+    "date": "2023-12-07T18:00:00+02:00",
+    "coverImage": "/banner.png",
```

Comment:
@erikwrede does this event have a banner?

Reply:
It will use the default https://graphql-events.org/banner.png if you are ok with that?

Reply:
@danstarns it's not my event, I just added it here for more visibility. Feel free to use the default.